src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -1048,9 +1050,15 @@ public static SyntaxKind GetPreprocessorKeywordKind(string text)
 
         public static IEnumerable<SyntaxKind> GetContextualKeywordKinds()
         {
-            for (int i = (int)SyntaxKind.YieldKeyword; i <= (int)SyntaxKind.WhenKeyword; i++)
+            ushort firstKeywordIdInclusive = (ushort)SyntaxKind.YieldKeyword;
+            ushort lastKeywordIdExclusive = (ushort)SyntaxKind.ElifKeyword;
+
+            for (ushort id = firstKeywordIdInclusive; id < lastKeywordIdExclusive; ++id)
             {
-                yield return (SyntaxKind)i;
+                if (Enum.IsDefined(typeof(SyntaxKind), id))
```

Comment:
> So we're not fixing windows that aren't broken :)

That's called *"economic growth"* 😆 Well, I for my part prefer having the computer do my work instead of *it* dictating me by comment what *I* have to do. 😁